IFC4x3/Templates/Object Composition/Element Voiding/Earthworks Cuttings/Documentation.md

```diff
@@ -0,0 +1,3 @@
+Earthworks Cuttings represent a voiding of an existing or constructed Earthen element (referred to as the voided element)  usually represented by a subtype of _IfcGeotechnicalElement_ or an _IfcEarthworksFill_, t. the _IfcEarthworksCut_ semantically represents the work of removing/excavating material from the voided element, and the associated geometrical representation is the resulting void created. A_IfcEarthworksCut_ May optionally be filled (fully or partially) with another element such as a foundation, earthen fill or foundation utilising the _IfcRelFillsElement_ relationship.
```

Comment:
You may want to revise `A_IfcEarthworksCut_ May`

Reply:
sorted